client/src/app/pages/questionnaires/questionnaires.tsx

```diff
@@ -0,0 +1,459 @@
+                          {t("dialog.title.download", {
+                            what: t("terms.YAMLTemplate"),
+                          })}
+                        </Button>
+                      </ToolbarItem>
+                    ) : null
+                  }
+                </ToolbarGroup>
+                <ToolbarItem {...paginationToolbarItemProps}>
+                  <SimplePagination
+                    idPrefix="questionnaire-table"
+                    isTop
+                    paginationProps={paginationProps}
+                  />
+                </ToolbarItem>
+              </ToolbarContent>
+            </Toolbar>
+            <Table {...tableProps} aria-label="Assement table">
+              <Thead>
+                <Tr>
+                  <TableHeaderContentWithControls {...tableControls}>
+                    <Th {...getThProps({ columnKey: "required" })} />
+                    <Th {...getThProps({ columnKey: "name" })} />
+                    <Th {...getThProps({ columnKey: "questions" })} />
+                    <Th {...getThProps({ columnKey: "rating" })} />
+                    <Th {...getThProps({ columnKey: "dateImported" })} />
+                  </TableHeaderContentWithControls>
+                </Tr>
+              </Thead>
+              <ConditionalTableBody
+                isLoading={isFetching}
+                isError={!!fetchError}
+                isNoData={currentPageItems.length === 0}
+                noDataEmptyState={
+                  <EmptyState variant="sm">
+                    <EmptyStateIcon icon={CubesIcon} />
+                    <Title headingLevel="h2" size="lg">
+                      No questionnaire available
+                    </Title>
+                    <EmptyStateBody>
+                      Use the import button above to add your questionnaire.
+                    </EmptyStateBody>
+                  </EmptyState>
+                }
+                numRenderedColumns={numRenderedColumns}
+              >
+                {currentPageItems?.map((questionnaire, rowIndex) => {
+                  return (
+                    <Tbody key={questionnaire.id}>
+                      <Tr>
+                        <TableRowContentWithControls
+                          {...tableControls}
+                          item={questionnaire}
+                          rowIndex={rowIndex}
+                        >
+                          <Td
+                            width={10}
+                            {...getTdProps({ columnKey: "required" })}
+                          >
+                            {questionnaire.required}
+                            <Switch
+                              id={`required-switch-${rowIndex.toString()}`}
+                              label="Yes"
+                              labelOff="No"
+                              isChecked={questionnaire.required}
+                              onChange={(_event, _checked) => {
```

Comment:
I have seen these params defined several different ways. I personally like using the underscore as a single char for params we aren't going to be using.

Reply:
I agree with you that looks better. 
Meanwhile _ can be used only once otherwise there is a conflict.

Reply:
So we aren't using either param?

Reply:
If we aren't using the params, I think you can just not pass them to the callback function here.